cmd/common_test.go

```diff
@@ -25,6 +25,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/spf13/cobra"
+
 	"github.com/mitchellh/go-homedir"
```

Comment:
does the `nsc` command always rely on the home directory?

Reply:
ah ok can be set via `$NSC_HOME`

Reply:
yes for path resolution - in various prompts you can enter ~/somepath if you are managing your keys by hand. Or for output files etc.